roles/update-access/test/user2

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Can we please name this file id_rsa_use2 (or something that indicates that it's the SSH private key)